app/services/user-summary.js

```diff
@@ -0,0 +1,54 @@
+import Ember from 'ember';
+
+const { get, set } = Ember;
+
+export default Ember.Service.extend({
+  store: Ember.inject.service(),
+  authentication: Ember.inject.service(),
+
+  weeekEntries: null,
+  monthEntries: null,
+
+  init() {
+    this._super(...arguments);
+    set(this, 'weekEntries', []);
+    set(this, 'monthEntries', []);
+  },
+
+  currentUserId: Ember.computed.reads('authentication.userId'),
+
+  currentUserIdChanged: Ember.observer('currentUserId', function() {
+    this.reload();
+  }),
```

Comment:
you said that you would have let this refresh into the components to prevent data reload when not displaying. As of long as this service exists I would keep the refresh into it, I would think that as services are always accessible they should also always contain the proper data. 
For my point of view, the solution to avoid data load when not displayed would be to keep the data into the component and get rid of this service. 
What do you think ? Is the actual code ok for now ?